### PR TITLE
[Feature] update calling buttons

### PR DESCRIPTION
**Pull request by David-Henner**
## What's new in this PR?

Call actions buttons can be confusing for some users because they don't feel very consistent. 
This PR updates the buttons so that their appearance best reflect the microphone, camera and speaker states within a call.
Buttons will appear as selected when the underlying state is turned on, and appear unselected when the state is off.
Additionally, the icons themselves reflect the state. i.e: if the microphone is off, the icon will have a strikethrough.

#### Todo

- [x] Update snapshot tests

#### Note

Don't be afraid by the number of files changed. It's mostly snapshots being updated

### CODE REVIEW

**Comment by David-Henner** [comment]
changing base branch to `develop` because we don't want the release of the updated buttons to depends on the status of large conference calls

File: Wire-iOS/Sources/UserInterface/Calling/CallActionsView/CallActionIconType.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    func icon(forState state: UIControl.State) -> StyleKitIcon {
+        switch state {
+        case .selected: return selectedIcon
+        default: return normalIcon
+        }
+    }
+
+    var label: String {
+        switch self {
+        case .microphone: return "voice.mute_button.title".localized
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/CallActionsView/CallActionIconType.swift:38]: Why not use the generated strings  `L10n.Localizable.Voice...`?

File: Wire-iOS/Sources/UserInterface/Calling/CallActionsView/VideoGridPresentationMode+CallActions.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+//
+
+import Foundation
+import WireSyncEngine
+
+extension VideoGridPresentationMode {
+    var title: String {
+        switch self {
+        case .activeSpeakers:
+            return "call.overlay.switch_to.speakers".localized
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/CallActionsView/VideoGridPresentationMode+CallActions.swift:26]: Why not use the generated strings  `L10n.Localizable.Call...`?

File: Wire-iOS/Sources/UserInterface/Calling/CallActionsView/CallActionIconType.swift (hunk 1 above)
Comment (David-Henner) [reply Wire-iOS/Sources/UserInterface/Calling/CallActionsView/CallActionIconType.swift:38]: Good catch! It was like this before and I missed it while refactoring

File: Wire-iOS/Sources/UserInterface/Calling/CallActionsView/VideoGridPresentationMode+CallActions.swift (hunk 2 above)
Comment (David-Henner) [reply Wire-iOS/Sources/UserInterface/Calling/CallActionsView/VideoGridPresentationMode+CallActions.swift:26]: Same same

**Review by KaterinaWire — APPROVED** [review]

**Pull request merged**